modules/openapi-generator/src/main/resources/go-server/model.mustache

```diff
@@ -19,6 +19,6 @@ const (
 type {{classname}} struct {
 {{#vars}}{{#description}}
 	// {{{description}}}{{/description}}
-	{{name}} {{^isEnum}}{{^isPrimitiveType}}{{^isContainer}}{{^isDateTime}}*{{/isDateTime}}{{/isContainer}}{{/isPrimitiveType}}{{/isEnum}}{{{dataType}}} `json:"{{baseName}}{{^required}},omitempty{{/required}}"{{#vendorExtensions.x-go-custom-tag}} {{{.}}}{{/vendorExtensions.x-go-custom-tag}}`
```

Comment:
This is the change that I'd like to discuss. In a previous version relative model objects were generated as classes. As we have `nullable` support now, I don't think we need to keep this implicit behavior. I'd rather expect to have the pointer defined as it should be -  using the `nullable` property.